store/CSDK/CarbonReader.cpp

```diff
@@ -19,15 +19,14 @@
 #include <jni.h>
 
 jobject CarbonReader::builder(JNIEnv *env, char *path, char *tableName) {
-
     jniEnv = env;
     jclass carbonReaderClass = env->FindClass("org/apache/carbondata/sdk/file/CarbonReader");
     jmethodID carbonReaderBuilderID = env->GetStaticMethodID(carbonReaderClass, "builder",
```

Comment:
If class path is wrong or class is not found in given path.  This will be NPE and jvm will crash.
Need to validate this and throw exception or return null with log to the user.

Reply:
I planed to handle exception in the future before. Can we raise a new PR for exception?

Reply:
No please handle at once. Because these are same feature. suggest we fix it and make it clean.  we should not merge PR which has issues as if some user uses it, he will get jvm crash

Reply:
I throw exception in build and handle it in main test case.